ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/RequestHandler.java

```diff
@@ -93,7 +89,8 @@ public KsqlEntityList execute(
               : Collections.emptyMap())
       );
       final ConfiguredStatement<?> configured = ConfiguredStatement.of(prepared,
-          SessionConfig.of(ksqlConfig, sessionProperties.getMutableScopedProperties())
+          SessionConfig.of(this.ksqlEngine.getKsqlConfig(),
```

Comment:
this is probably not the right place to generate a `ConfiguredStatement` anymore, because the configuration can change between this line and the point where the command is actually executed from `executeStatement`. We should instead leave this as a `PreparedStatement` and assemble the final `ConfiguredStatement` either after `commandQueueSync.waitFor` or in the executor implementation itself (the latter would require a bit more refactoring).